contents/swift/welcome-to-swift.md

```diff
@@ -13,46 +13,50 @@
+Authors: [Ch'ng Ming Shin](https://github.com/ablyx/cs3281-website/blob/mingshin-week6-progress/students/AY1617S2/ch'ngMingShin/Ch'ngMingShin-Resume.md), [Yong Zhi Yuan](https://github.com/Zhiyuan-Amos), [Jiang Chunhui](https://github.com/Adoby7)
 
-# Overview
+## What is Swift?
 
-Welcome to Swift, THE preferred language to do iOS programming. Introduced in 2014, Swift is a fast, modern and safe programming language which supports playgrounds, an innovative feature that allows programmers to experiment with Swift code and see the results immediately, without the overhead of building and running an app.
+Swift is a fast, modern and safe programming language mainly used for iOS programming.
+Introduced in 2014, Swift has concise and expressive syntax, especially when compared with its predecessor language **Objective-C**. On top of that, Swift also supports playgrounds, a feature that allows programmers to experiment with Swift code and see the results immediately (without the overhead of building and running an app).
 
-# Getting Started
+## Why should I learn Swift?
 
-Reading the [Language Guide in the official documentation](https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/) is definitely recommended, since it explains everything quite clearly, albeit quite verbose.
+1. **Learning Swift can help you land your dream career.**
+Swift is mainly used for iOS development and iOS developers are well paid. The [salary statistics](https://www.indeed.com/salaries/iOS-Developer-Salaries) shows that iOS developer in US can earn more than $120K per year. In addition, mobile application development is also becoming more popular as of recent times. A [survey in StackOverFlow](https://insights.stackoverflow.com/survey/2017#developer-profile-developer-type) shows that mobile development, although lesser than web development, still takes up a large share of the market.
 
-If you're just looking to take a dip into Swift, here are the [basics](https://guides.codepath.com/ios/Swift-Basics) you'll need to understand.
+1. **Swift is easy to learn.** Swift is designed to be a simple and user-friendly language by Apple. Its syntax resembles English, making it easy to read and understand. Any programmer who is familiar with Python, Java or C++ can learn Swift quickly! Even if you are not familiar with any language, Swift is an excellent language to start out with.
 
-And if you are really pressed for time, here's a couple of cheatsheets with code examples:
+1. **Swift is an [open source project](https://github.com/apple/swift).** Learning about Swift and contributing to the language may be a useful experience for those interested. Apart from learning about Swift from its documentation, you can also dig into the source code to get new insights. Typically, you can start submitting issues or patches if you find there is any way to improve the language!
```

Comment:
As this arcticle is for new-comers to swift, will it be too hard for them to directly dive into the source code? As majority of them should have no background in programing language implementation. It will be hard to learn something practical from it.

Reply:
Yes. I just want to give them an impression that Swift is open source, as it is "good to know" infomation. Readers may choose to dive after they have learnt some knowledge.